src/plugins/decor/ui/modals/ChangeDecorationModal.tsx

```diff
@@ -187,7 +222,7 @@ function ChangeDecorationModal(props: ModalProps) {
                     }}
                     getItemKey={item => typeof item === "string" ? item : item.hash}
                     getSectionKey={section => section.sectionKey}
-                    renderSectionHeader={section => <SectionHeader section={section} />}
+                    renderSectionHeader={section => <SectionHeader section={section} presets={presets} />}
```

Comment:
instead of passing all presets and doing a .some, why not just pass the activeDecorationPreset variable?

```suggestion
                    renderSectionHeader={section => <SectionHeader section={section} activePreset={activeDecorationPreset} />}
```